node/totalStakedAPI/interface.go

```diff
@@ -0,0 +1,9 @@
+package totalStakedAPI
+
+import "math/big"
+
+// TotalStakedValueHandler defines the behavior of a component able to return total staked value
+type TotalStakedValueHandler interface {
```

Comment:
Interfaces should be placed where are used, not where are implemented. Plese move this in node/external package

Reply:
done.